src/react-apps/applications/shared/src/file-editor/FileEditor.tsx

```diff
@@ -144,17 +181,43 @@ class FileEditor extends React.Component<IFileEditorProvidedProps, IFileEditorSt
     this.loadFileContent(fileName);
   }
 
-  public saveFile = () => {
+  public saveFile = async (e: any) => {
     const altinnWindow: IAltinnWindow = window as IAltinnWindow;
-    const { org, service} = altinnWindow;
+    const { org, service } = altinnWindow;
     const servicePath = `${org}/${service}`;
     const postUrl = `${altinnWindow.location.origin}/designer/${servicePath}/ServiceDevelopment` +
-    `/SaveServiceFile?fileEditorMode=${this.props.mode}&fileName=${this.state.selectedFile}`;
-    post(postUrl, this.state.value, {headers: {'Content-type': 'text/plain;charset=utf-8'}}).then(() => {
-      if (this.state.mounted && this.props.closeFileEditor) {
-        this.props.closeFileEditor();
-      }
+      `/SaveServiceFile?fileEditorMode=${this.props.mode}&fileName=${this.state.selectedFile}`;
+
+    const saveRes: any = await post(postUrl, this.state.value, {
+      headers: {
+        'Content-type': 'text/plain;charset=utf-8',
+      },
     });
+
+    if (saveRes.isSuccessStatusCode === false) {
+      console.log('save error', saveRes);
```

Comment:
`console.error` ?

Reply:
Modified all related console.log to console.error.